src/Microsoft.TestPlatform.ObjectModel/RunSettings/RunConfiguration.cs

```diff
@@ -524,12 +572,19 @@ public static RunConfiguration FromXml(XmlReader reader)
                             break;
 
                         default:
-                            throw new SettingsException(
```

Comment:
Isn't this a change in behavior.
How do we notify the user that something was ignored or is not supported ?